tests/acceptance/guid-node/addons-test.ts

```diff
@@ -0,0 +1,342 @@
+        // Edit first account display name
+        await fillIn('[data-test-display-name-input]', 'New S3 Account Display Name');
+        assert.dom('[data-test-root-folder-save]').isNotDisabled('Save button is enabled after displayName change');
+        await fillIn('[data-test-display-name-input]', s3AccountsDisplayNamesAndRootFolders[0].displayName);
+        assert.dom('[data-test-root-folder-save]').isDisabled('Save button is disabled after displayName reset');
+        // Edit first account root folder
+        await click('[data-test-folder-link]:first-child');
+        assert.dom('[data-test-folder-path-option]').exists({ count: 2 }, '2 folders in path');
+        await click('[data-test-root-folder-option]:first-child');
+        assert.dom('[data-test-root-folder-save]')
+            .isNotDisabled('Save button is enabled after selecting new root folder option');
+        await fillIn('[data-test-display-name-input]', 'New S3 Account Display Name');
+        // Save with root folder change and display name change
+        await click('[data-test-root-folder-save]');
+
+        // Check that changes were saved
+        assert.dom('[data-test-addons-tab-all-addons]')
+            .hasAttribute('aria-selected', 'true', 'Back to all addons tab after saving changes');
+        await click('[data-test-addons-tab-connected-accounts]');
+        await click('[data-test-addon-card="Amazon S3"] [data-test-addon-card-configure]');
+        assert.dom('[data-test-edit-connected-location]').exists({ count: 2 }, 'Two editable accounts are still there');
+        assert.dom('[data-test-remove-connected-location]').exists({ count: 2 }, 'Two remove buttons are present');
+        const firstDisplayName = document.querySelectorAll('[data-test-configured-addon-name]')[0];
+        const firstRootFolder = document.querySelectorAll('[data-test-configured-addon-root-folder]')[0];
+        assert.dom(firstDisplayName)
+            .containsText('New S3 Account Display Name', 'Display name change is saved');
+        assert.dom(firstRootFolder).containsText('s3root-1-1', 'Root folder change is saved');
+
+        // Remove other account
+        await click('[data-test-remove-connected-location]:last-child');
+        assert.dom('[data-test-confirm-remove-connected-location-button]').exists('Confirm remove button is present');
+        assert.dom('[data-test-cancel-remove-connected-location-button]').exists('Cancel remove button is present');
+        await click('[data-test-confirm-remove-connected-location-button]');
+        assert.dom('[data-test-edit-connected-location]')
+            .exists({ count: 1 }, 'One editable account is left after removing one');
+        assert.dom('[data-test-remove-connected-location]')
+            .exists({ count: 1 }, 'One remove button is present after removing one');
+    });
+
+    // Skip until issue with added account not showing up is resolved
+    skip('Adding new configured addons', async function(assert) {
```

Comment:
Don't forget about this one.

Reply:
Or should we just try to fix it now?

Reply:
I have a ticket to fix this in a separate issue. I've made a note for myself in that ticket to un-skip this test in that ticket